addons/event/models/event_event.py

```diff
@@ -395,17 +397,17 @@ def _compute_event_registrations_sold_out(self):
     def _compute_date_begin_tz(self):
         for event in self:
             if event.date_begin:
-                event.date_begin_located = format_datetime(
-                    self.env, event.date_begin, tz=event.date_tz, dt_format='medium')
+                event.date_begin_located = format_date(self.env, event.date_begin.date, date_format='long') \
+                                         + format_time(self.env, event.date_begin.time, tz=event.date_tz, time_format='short')
             else:
                 event.date_begin_located = False
 
     @api.depends('date_tz', 'date_end')
     def _compute_date_end_tz(self):
         for event in self:
             if event.date_end:
-                event.date_end_located = format_datetime(
-                    self.env, event.date_end, tz=event.date_tz, dt_format='medium')
+                event.date_end_located = format_date(self.env, event.date_end.date, date_format='long') \
+                                       + format_time(self.env, event.date_end.time, tz=event.date_tz, time_format='short')
             else:
                 event.date_end_located = False
```

Comment:
Actually where are these fields used? I wonder if we could not use the widget again, if they are used in templates again, instead of doing the change :) The issue being that the format_date does not really handle with timezone, so we must first localize it before using format_date. ( That's why using format_datetime for both date and time is so powerful :p )